sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java

```diff
@@ -155,8 +167,21 @@ public SentryFrameMetricsCollector(
           lastFrameStartNanos = startTime;
           lastFrameEndNanos = startTime + cpuDuration;
 
+          // Most frames take just a few nanoseconds longer than the optimal calculated
+          // duration.
+          // Therefore we subtract one, because otherwise almost all frames would be slow.
+          final boolean isSlow = cpuDuration > oneSecondInNanos / (refreshRate - 1);
```

Comment:
@stefanosiano I moved this over from `AndroidProfiler` but I'm not sure if this is correct:
We're using the `cpuDuration` here, shouldn't it be `totalFrameduration`?

Reply:
cpu duration is fine, we're doing the same of google's JankStats
the cpuDuration `Return time spent on the main thread (cpu) for frame creation. It doesn't consider time spent on the render thread (gpu).`